docs/contributing/contributor-compensation.md

```diff
@@ -2,25 +2,38 @@
+* Corporate contractors are companies that do work for Decred, e.g. Company 0, Raedah, Ditto. Corporate contractors typically employ a number of contributors who work on Decred (but for the corporate contractor).
+* Independent contractors are people that do work for Decred. This [page](advanced/contributing-to-Decred) explains how people who contribute to the project can become independent contractors.
 
-The RFP process worked well for certain types of tasks, however a need arose to have contributors engaged and compensated over the longer term and on a more flexible basis[^2]. In January 2017 the RFP system was changed to a contractor model and Decred began to hire it's first contractors.
+Contractors contribute in a wide variety of ways including software development, design, communications, community support, research and documentation.
+ 
+Decred currently has over 50 contributors (independant contractors and employees of corporate contractors), and you can see them on [decred.org/contributors](https://decred.org/contributors/) in artfully styled portraits with badges indicating whether they work for a corporate contractor.
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/User.svg" /> Contractors
+## <img class="dcr-icon" src="/img/dcr-icons/Wallet.svg" /> Billing and payments
+
+At the end of each month, each contractor prepares an invoice and submits it to the _Decred Holdings Group_ LLC (DHG). The invoice lists the items the contractor has worked on and a charge (in $USD) for each - typically hours worked charged at an agreed hourly rate.
+
+$USD amounts are converted to DCR at the average daily market rate for the previous month. 
 
-Decred currently has over 50 active contractors and they can be viewed on the decred.org [Contributors](https://decred.org/contributors) page. The active contractors are made up of both individuals and corporate contractors. Contractors contribute in a wide variety of ways including software development, design, marketing, community management and documentation.
+Once all of the invoices have been received and processed, payments to all contractors are made on the same day. This typically happens in the middle of the following month, around 2-4 weeks in. The intention is to turn payments around as quickly as possible but the process takes time and relies on timely invoicing by contractors. 
 
-Contractors submit a monthly invoice to DHG (Decred Holding Group) and are paid in Decred according to an hourly rate.
+As the USD/DCR conversion rate is pegged to the previous month, contractors are exposed to any volatility in the price of DCR that occurs during this period. This (somewhat incidentally) enhances the degree to which contractors have skin in the game, and makes financial planning both necessary and interesting.
```

Comment:
As I mentioned [in the comments](https://github.com/decred/dcrdocs/issues/783#issuecomment-455777301) I don't think this paragraph really belongs in the docs. I will bow to the will of the majority here though, I don't feel too strongly about it. If the general consensus thinks it should go there, lets include it.

Reply:
I'm on the fence here too. It goes beyond the usual remit of docs as I see it, but I can see why at least the first sentence might be helpful for new contractors.

Would it be correct to say that @s-ben  and @xaur are keen for this to be included?

Reply:
My idea was to set proper expectations.

There are two parts of the volatility risk period. First is the period between first and last day of the month the work was done (and billed for). Volatility in this period affects the rate at which payment is calculated. This is what one subscribes for and more or less expects. Second period is between last day of the worked/billed month and the date of payment on the next month. Volatility in this period (specifically, the price at the moment of payment) affects the rate at which the contractor can liquidate his DCR. I personally found the second part more subtle and tricky, hence suggesting to clarify it.

It's a bit unclear to me how exactly and which of the periods (first, second or both) increase skin in the game. I agree it makes financial planning necessary. Not sure about "interesting". It can be both exciting/interesting and challenging/stressful.

Reply:
Late to the comment party, but glad to see this paragraph is still in. I think it will help set better expectations. 

As for the first and second periods, presumably the second period is also part of "skin in the game", because when the invoices are being paid, DHG could simply just adjust the payment so the USD equivalent at the time of payment was equal to what it would have been when the invoice was submitted, taking on the volatility itself between invoicing and payment. As such, I think it would be more accurate to say something like, "contractors are exposed to volatility from the invoiced period AND the time between invoice and payment". I think the the text is good to go as is. Just wanted to put in my 2 atoms, as this is going to be an issue moving forward as the DAE scales. Skin in the game is an important motivator, but we should also be aware that there will be people who are highly ideologically motivated, financially savvy, and simply not have the capitol to take on that kind of volatility. I am close to that group myself actually. My first two invoices I had to liquidate most of my DCR for ~50% and ~40 less than my USD rate. Super bad timing, and I was able to weather it. But many will not.